utils.py

```diff
@@ -453,3 +454,30 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+cache_slug = None  # pylint: disable=invalid-name
```

Comment:
it is kind of constant, maybe upper case (i.e CACHE_SLUG), after that we won't need 'pylint: disable=invalid-name'